tests/bluetooth/tester/src/audio/btp_csis.c

```diff
@@ -69,9 +69,23 @@ static uint8_t csis_sirk_type(const void *cmd, uint16_t cmd_len, void *rsp, uint
 {
 	const struct btp_csis_sirk_type_cmd *cp = cmd;
 
-	enc_sirk = cp->encrypted != 0U;
-
-	LOG_DBG("SIRK type: %s", enc_sirk ? "encrypted" : "plain text");
+	switch (cp->type) {
+		case 0:
+			LOG_DBG("SIRK type: plain text");
+			sirk_read_response = BT_CSIP_READ_SIRK_REQ_RSP_ACCEPT;
+			break;
+		case 1:
+			LOG_DBG("SIRK type: encrypted");
+			sirk_read_response = BT_CSIP_READ_SIRK_REQ_RSP_ACCEPT_ENC;
+			break;
+		case 2:
```

Comment:
Instead of magic numbers, we should define the possible values (both here and in AutoPTS)